src/lib/config/passport.ts

```diff
@@ -104,8 +108,8 @@ export function hasRole(role: string) {
 }
 
 export function logout(req: express.Request, res: express.Response) {
-	strategy.logout(req, (err, url) => {
-		req.logout()
-		res.redirect(url)
-	})
+	// strategy.logout(req, (err, url) => {
+	// 	req.logout()
+	// 	res.redirect(url)
+	// })
```

Comment:
What's the deal here? No longer supported? If there's no need to do a logout on the strategy can we still do a `req.logout()` and `res.redirect(url)`?